CMakeLists.txt

```diff
@@ -103,9 +110,9 @@ else()
 
   if(TRITON_ENABLE_GPU)
     rapids_cuda_init_architectures(RAPIDS_TRITON_BACKEND)
-    project(RAPIDS_TRITON_BACKEND VERSION 21.12.00 LANGUAGES CXX CUDA)
+    project(RAPIDS_TRITON_BACKEND VERSION 22.02.00 LANGUAGES CXX CUDA)
```

Comment:
We need to update this to `22.04` (and a few other places).

Reply:
This is the version of the FIL backend, which is going to be updated to `22.03` this release

Reply:
Actually never mind. Didn't realize Rapids-Triton was following its own release cycle.